scripts/tests/cirque_tests.sh

```diff
@@ -44,45 +53,73 @@ function __virtual_thread_clean() {
     vthread_pid=$(ps aux | grep "[o]t-ncp-ftd" | awk '{print $2}')
     if [ ! -z "$vthread_pid" ]; then
         for pid in "$vthread_pid"; do
-            kill -2 "$pid"
+            sudo kill -2 "$pid"
         done
     fi
 }
 
-__cirquetest_start_flask() {
+function __cirquetest_start_flask() {
+    echo "Start Flask"
     cd "$REPO_DIR"/third_party/cirque/repo
     sudo FLASK_APP='cirque/restservice/service.py' \
         PATH="$PATH":"$REPO_DIR"/third_party/cirque/repo/openthread/output/x86_64-unknown-linux-gnu/bin/ \
-        python3 -m flask run
+        python3 -m flask run >/dev/null 2>&1
 }
 
-__cirquetest_clean_flask() {
+function __cirquetest_clean_flask() {
+    echo "Cleanup Flask"
     __flask_clean
     __socat_clean
     __virtual_thread_clean
 }
 
-cirquetest_bootstrap() {
+function cirquetest_bootstrap() {
     set -e
     cd "$REPO_DIR"/third_party/cirque/repo
     sudo apt-get install -y bazel socat psmisc tigervnc-standalone-server tigervnc-viewer python3-pip python3-venv python3-setuptools libdbus-glib-1-dev libgirepository1.0-dev
     pip3 install pycodestyle==2.5.0 wheel
     make NO_GRPC=1 install -j
     ./dependency_modules.sh
     pip3 install -r requirements_nogrpc.txt
+
+    # Call activate here so the later tests can be faster
+    # set -e will cause error if activate.sh is sourced twice
+    # this is an expected behavior caused by pigweed/activate.sh
+    set +e
+    source "$REPO_DIR/scripts/bootstrap.sh"
+    source "$REPO_DIR/scripts/activate.sh"
 }
 
-cirquetest_run_test() {
+function cirquetest_run_test() {
     # Start Cirque flash server
     __cirquetest_start_flask &
     sleep 5
-    cd "$REPO_DIR"/src/test_driver/linux-cirque
-    ./do_on-off-cluster-test.sh
-    exit_code=$?
+    cd "$TEST_DIR"
+    ./"$1.sh"
+    exitcode=$?
     __cirquetest_clean_flask
     # Cleanup containers and possibly networks to run more tests
-    yes | docker system prune
-    return "$exit_code"
+    echo "Do docker system prune"
+    (yes | docker system prune) >/dev/null 2>&1
```

Comment:
Why?

Reply:
Just do cleanups and provide a clean environment for later tests.

Reply:
please just prune the dockers created by cirque, not touch the dockers created by others.

Reply:
How can I know if a docker container is created by cirque?